.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,7 +37,7 @@ Please review the following and check all that apply:
 
 - [ ] I have reviewed the guidelines for [How to Contribute](https://github.com/apache/solr/blob/main/CONTRIBUTING.md) and my code conforms to the standards described there to the best of my ability.
 - [ ] I have created a Jira issue and added the issue ID to my pull request title.
-- [ ] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
+- [ ] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended, not available for branches on forks living under an organisation)
```

Comment:
Is this clear and/or would it be useful enough to include in the template?

On the https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork docs it says

> Who can use this feature?
> People with push access to the upstream repository of a fork in a personal account can commit to the forked branches.

and _"fork in a personal account"_ is the subtle bit it seems.

https://github.com/orgs/community/discussions/5634 is related.